[2025-04-03T13:26:47.197+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-03T13:26:47.227+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.create_tables_task manual__2025-03-26T13:33:39.761724+00:00 [queued]>
[2025-04-03T13:26:47.235+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.create_tables_task manual__2025-03-26T13:33:39.761724+00:00 [queued]>
[2025-04-03T13:26:47.236+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2025-04-03T13:26:47.268+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): create_tables_task> on 2025-03-26 13:33:39.761724+00:00
[2025-04-03T13:26:47.273+0000] {standard_task_runner.py:63} INFO - Started process 170 to run task
[2025-04-03T13:26:47.276+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'create_tables_task', 'manual__2025-03-26T13:33:39.761724+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpwe3w0ve4']
[2025-04-03T13:26:47.279+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask create_tables_task
[2025-04-03T13:26:47.344+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_pipeline.create_tables_task manual__2025-03-26T13:33:39.761724+00:00 [running]> on host 7051e0ca121e
[2025-04-03T13:26:47.421+0000] {abstractoperator.py:725} ERROR - Exception rendering Jinja template for task 'create_tables_task', field 'sql'. Template: '/opt/sql/create_table.sql'
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/abstractoperator.py", line 717, in _do_render_template_fields
    rendered_content = self.render_template(
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/template/templater.py", line 166, in render_template
    template = jinja_env.get_template(value)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/environment.py", line 1016, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/loaders.py", line 209, in get_source
    raise TemplateNotFound(
jinja2.exceptions.TemplateNotFound: '/opt/sql/create_table.sql' not found in search path: '/opt/airflow/dags'
[2025-04-03T13:26:47.451+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2479, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2633, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context, jinja_env=jinja_env)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 3094, in render_templates
    original_task.render_template_fields(context, jinja_env)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 1337, in render_template_fields
    self._do_render_template_fields(self, self.template_fields, context, jinja_env, set())
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/abstractoperator.py", line 717, in _do_render_template_fields
    rendered_content = self.render_template(
                       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/template/templater.py", line 166, in render_template
    template = jinja_env.get_template(value)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/environment.py", line 1016, in get_template
    return self._load_template(name, globals)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/jinja2/loaders.py", line 209, in get_source
    raise TemplateNotFound(
jinja2.exceptions.TemplateNotFound: '/opt/sql/create_table.sql' not found in search path: '/opt/airflow/dags'
[2025-04-03T13:26:47.477+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=etl_pipeline, task_id=create_tables_task, run_id=manual__2025-03-26T13:33:39.761724+00:00, execution_date=20250326T133339, start_date=20250403T132647, end_date=20250403T132647
[2025-04-03T13:26:47.560+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 133 for task create_tables_task ('/opt/sql/create_table.sql' not found in search path: '/opt/airflow/dags'; 170)
[2025-04-03T13:26:47.569+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-04-03T13:26:47.592+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-03T13:26:47.615+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
